core/domain/obj_services_test.py

```diff
@@ -63,3 +64,43 @@ def test_object_js_filenames(self):
                 self.assertIn(
                     'oppia.directive(\'%s\'' % normalized_directive_name,
                     template, msg='(%s)' % obj_type)
+
+
+class ObjectDefaultValuesUnitTests(test_utils.GenericTestBase):
+    """Test that the default value of objects recorded in
+    extensions/interactions/object_defaults.json correspond to
+    the defined default values in objects.py for all objects that
+    are used in rules."""
+
+    def test_all_rule_input_fields_have_default_values(self):
+        """Checks that all rule input fields have a default value, and this
+        is provided in get_default_values()."""
```

Comment:
Ditto for this docstring (please make sure ``"""`` is on its own line and does not have whitespace between it and the first statement.